src/test/kotlin/test/tests/hashers/CommitHasherTest.kt

```diff
@@ -199,5 +210,66 @@ class CommitHasherTest : Spek({
         }
     }
 
+    given("test of commits date facts") {
+        val testRepoPath = "../testrepo1"
+        val testRepo = TestRepo(testRepoPath)
+        val author1 = Author("Test1", "test@domain.com")
+        val author2 = Author("Test2", "test@domain.com")
+
+        val repo = Repo(rehash = "rehash", commits = listOf())
+        val mockApi = MockApi(mockRepo = repo)
+        val facts = mockApi.receivedFacts
+
+        afterEachTest {
+            facts.clear()
+        }
+
+        it("send two facts") {
+            testRepo.createFile("test1.txt", listOf("line1", "line2"))
+            testRepo.commit(message = "initial commit",
+                            author = author1,
+                            date = createDate(month = 1, day = 1,  // Sunday.
```

Comment:
might be worth to point year explicitly, if createDate() gets changed for some reason, then a failing test might take by surprise

Reply:
Ok.